config/staging/iatlas-runner.yaml

```diff
@@ -0,0 +1,16 @@
+template_path: gitlab-runner.yaml
+stack_name: iatlas-gitlab-runner-staging
+dependencies:
+  - common/iatlasvpc.yaml
+  - staging/iatlas-db.yaml
+  - staging/iatlas-kms.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  InstanceType: "t2.nano"
+  VpcName: iatlasvpc
```

Comment:
can you use the sceptre !stack_output reference instead?

Reply:
That name actually becomes two stack output references: a security group output and a VpcSubnet, so it's shorter to input just a single VpcName.